src/docs/reference/modules/Media.Slugify/README.md

```diff
@@ -0,0 +1,16 @@
+# Media Slugify (`OrchardCore.Media.Slugify`)
+
+The Media Slugify feature enables _slugifying_ assets in the Media Library.
+
+## Purpose
+
+By default the Media Library does not restrict the naming of folders and files. For example having a file `The team (2020).jpg` in folder `Images & docs` is allowed. The URL of this asset would be `/media/Images%20&%20docs/The%20team%20(2020).jpg`.
+
+This obviously is not a very SEO-friendly URL.
```

Comment:
This obviously not
or
This is obviously not

Reply:
https://github.com/OrchardCMS/OrchardCore/pull/7729